platform/lib/sdk/src/main/kotlin/io/hamal/lib/sdk/api/Namespace.kt

```diff
@@ -44,7 +41,8 @@ data class ApiNamespaceList(
 
 data class ApiNamespace(
     val id: NamespaceId,
-    val name: NamespaceName
+    val name: NamespaceName,
+    val features: NamespaceFeatures = NamespaceFeatures.default
```

Comment:
no default here - please replace this later when you add this to Namespace domain object